### ISSUE TITLE
Make an auto-complete list of available solar systems

### ISSUE THREAD

**Issue by d-olivaw** [d-olivaw/eveWantIsk#2]
We will need to figure out how to access the list of solar systems from CREST (depends on issue #1).

**Comment by d-olivaw** [comment]
See the comment in #3, it looks like the solar system information may not be directly available from CREST

**Comment by d-olivaw** [comment]
goals have changed, and this issue is no longer needed
